src/coreclr/jit/compiler.h

```diff
@@ -1798,26 +1797,50 @@ struct fgArgTabEntry
         return m_byteOffset;
     }
 
-    void SetByteSize(unsigned byteSize, unsigned byteAlignment)
+    void SetByteSize(unsigned byteSize, bool isStruct, bool isFloatHfa)
     {
-        assert(byteAlignment != 0);
-        const unsigned alignedByteSize = roundUp(byteSize, byteAlignment);
+
+#ifdef OSX_ARM64_ABI
+        unsigned roudnedByteSize;
```

Comment:
Typo (here and below)
```suggestion
        unsigned roundedByteSize;
```